gettingStarted.md

```diff
@@ -0,0 +1,354 @@
+
+```typescript
+core.addPlugins([
+  AddressSearch(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.TOP_LEFT, // Position of the plugin on the map
+      addLoading: 'plugin/loadingIndicator/addLoadingKey',
+      removeLoading: 'plugin/loadingIndicator/removeLoadingKey',
+    })
+  ),
+  iconMenu,
+  Export(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.BOTTOM_LEFT, // Position of the export plugin
+    })
+  ),
+  LoadingIndicator(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.MIDDLE_MIDDLE, // Position of the loading indicator
+    })
+  ),
+  Legend({
+    displayComponent: true,
+    layoutTag: NineLayoutTag.BOTTOM_RIGHT, // Position of the legend
+  }),
+  Scale(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.BOTTOM_RIGHT, // Position of the scale
+    })
+  ),
+  Toast(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.BOTTOM_MIDDLE, // Position of the toast messages
+    })
+  ),
+  Pins({
+    appearOnClick: { show: true, atZoomLevel: 6 }, // Configuration for the pins on the map
+    coordinateSource: 'plugin/addressSearch/chosenAddress',
+    toastAction: 'plugin/toast/addToast',
+  }),
+  Gfi(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.MIDDLE_LEFT, // Position of the GFI plugin
+    })
+  ),
+  ReverseGeocoder({
+    url: 'https://geodienste.hamburg.de/HH_WPS',
+    addLoading: 'plugin/loadingIndicator/addLoadingKey',
+    removeLoading: 'plugin/loadingIndicator/removeLoadingKey',
+    coordinateSource: 'plugin/pins/transformedCoordinate',
+    addressTarget: 'plugin/addressSearch/selectResult',
+    zoomTo: 7,
+  }),
+]);
+```
+
+## 5. Integrating Plugins into mapConfiguration.ts
+
+The `mapConfiguration.ts` file serves as the central configuration point for your map application, providing a comprehensive structure for defining how your map should look and behave. This file, in combination with `addPlugins.ts` and `polar-client.ts`, integrates and manages various plugins within the POLAR framework.
+
+### 5.1 Basic Structure of mapConfiguration.ts
+
+Initially, the `mapConfiguration.ts` file starts with foundational settings for the map, such as projection systems, language settings, and the theme configuration using Vuetify. It also sets up layers, address search, drawing options, and other components that influence how the map and its plugins function. Here's an excerpt of the core configuration:
+
+``` Typescript
+export const mapConfiguration = {
+  language: 'en',
+  epsg: 'EPSG:25832',
+  locales: language,
+  vuetify: {
+    theme: {
+      themes: {
+        light: {
+          primary: somewhatBlue,
+          primaryContrast: notQuiteWhite,
+          secondary: eigengrau,
+          secondaryContrast: notQuiteWhite,
+        },
+      },
+    },
+  },
+  // Other map configurations like namedProjections, addressSearch, layers, etc.
+};
+```
+
+### 5.2 Adding Plugins Using addPlugins.ts
```

Comment:
A similar part has been added above leading to some duplication. However, I'm not yet sure where to merge them. How would you approach this?

Reply:
I merged some parts and think the new structures is more clearer. What do you think?

Reply:
done